server.py

```diff
@@ -32,6 +33,21 @@
 })
 
 
+def check_default_env_vars():
+
+    env_vars = ["MONGODB_URL", "RABBIT_CS_QUEUE", "RABBIT_CTP_QUEUE", "RABBIT_CORA_QUEUE",
+                "RABBITMQ_HOST", "RABBITMQ_PORT", "RABBITMQ_DEFAULT_USER", "RABBITMQ_DEFAULT_PASS",
+                "RABBITMQ_DEFAULT_VHOST", "RABBITMQ_HOST2", "RABBITMQ_PORT2"]
+
+    for i in env_vars:
```

Comment:
This would be more pythonic:

```python
def _get_value(key):
    value = os.getenv(key)
    if not value:
        raise ValueError("No value set for " + key)
    else:
        return value

try:
    for i in env_vars:
        _get_value(i)
 except ValueError as e:
    logger.error("Unable to start service - envvar not set", error=e)